pkg/controller/machine/controller.go

```diff
@@ -163,7 +163,7 @@ func (r *ReconcileMachine) Reconcile(request reconcile.Request) (reconcile.Resul
 		}
 
 		if !r.isDeleteAllowed(m) {
-			klog.Infof("Skipping reconciling of machine %q", name)
+			klog.Infof("Deleting machine %q is not allowed", name)
```

Comment:
or may "Deleting controller machine node is not allowed. Skipping reconciliation of this machine"

Reply:
I'd suggest "Deleting machine hosting this controller is not allowed. Skipping reconciliation of machine %q", that way we are clearly saying why it is being prevented and still explicitly calling out the machine name